.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: 'bundler'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+      time: '08:00'
+      timezone: America/New_York
+    open-pull-requests-limit: 3
```

Comment:
Any reason not to have them just get created in real-time as they come up?

Reply:
Good question, I don't have a good one, will remove the schedule (will keep the open prs limit so we don't get flooded with more PRs than we can merge

Reply:
done!